app/views/hackdays/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>HackDays!</h1>
+<% @hackdays.sort_by {|hackday| hackday.start}.reverse.each do |hackday| %>
```

Comment:
this sort of logic should be:

a) done in the controller
b) should be done in the sql side rather than in the ruby side.